src/components/onboarding/steps.js

```diff
@@ -197,13 +197,6 @@ export const steps = t => ([
     position: 'right',
     style: styles.step,
   },
-  {
-    title: t('Explore the network'),
-    text: t('View all the transactions happening on the Lisk Network in real time. Search for addresses and transactions.'),
-    selector: '#explorer',
-    position: 'right',
-    style: styles.step,
-  },
```

Comment:
This step could be re-assigned to the "search input field". Than it should be before "Dashboard" and  update the second sentence to `Search for delegates, addresses and transactions.`